generation.py

```diff
@@ -1,7 +1,11 @@
+import time
 from random import randint
 
 def generate_matrix(width, height):
+    start = time.time()
```

Comment:
Why do you need measure performance of matrix generation, we discussed only performance measurement of  matrix multiplication method   , is not it?